### ISSUE TITLE
Best wishes for your life, Gold

### ISSUE THREAD

**Issue by PSGitHubUser1** [Goldorion/MCreatorFabricGenerator#542]
I really liked about how you did not left us till now, Despite the things you faced.
Thanks for your big contribution in MCreator community.

If you don't know what I am saying then see [this](https://mcreator.net/forum/107531/final-chapter-my-journey).

**Comment by Goldorion** [comment]
Thank you 🫡

**Comment by PSGitHubUser1** [comment]
> Thank you 🫡

Gold, 

I am feeling really shameful by asking this.. but who is going to manage your all projects like FBGen?

**Comment by Realbizzy** [comment]
> > Thank you 🫡
> 
> Gold,
> 
> I am feeling really shameful by asking this.. but who is going to manage your all projects like FBGen?

she said nerdy is gonna take over some of her plugins!

**Comment by PSGitHubUser1** [comment]
> > > Thank you 🫡
> > 
> > Gold,
> > 
> > I am feeling really shameful by asking this.. but who is going to manage your all projects like FBGen?
> 
> she said nerdy is gonna take over some of her plugins!

'Her'? what?

its He/Him .

His old dp or pfp had a guy (brown hair) with a smiling face.

**Comment by Realbizzy** [comment]
> > > > Thank you 🫡
> > > 
> > > 
> > > Gold,
> > > I am feeling really shameful by asking this.. but who is going to manage your all projects like FBGen?
> > 
> > 
> > she said nerdy is gonna take over some of her plugins!
> 
> 'Her'? what?
> 
> its He/Him .
> 
> His old dp or pfp had a guy (brown hair) with a smiling face.

sorry i confuse lots of peoples pronouns, no matter their pfp or name xd

**Comment by PSGitHubUser1** [comment]
Its been 4 months. I am waiting for new update! We still don't know who's gonna update the repo.

@Goldorion Sorry for the ping, even if you are not the  'main' maintainer now. Do you know who's gonna run this repo? Is it Nerdy? or the Klemen himself? 👀

**Comment by Goldorion** [comment]
Nobody decided to take it back for the moment. Nerdy didn't want to continue when we talked about my plugins (at least not soon). Concerning Klemen, I already asked him multiple times if there was a possibility to make it official in some way, but the answer always was the same. The amount of active contributors for the core is not high enough to justify the addition of another big generator. In addition, I was the only one in the team of main contributors to know and work with Fabric, so Klemen would need to learn Fabric before taking it back. Another reason was also the fact the generator, even though I worked hard on it and improved it quite a lot since the beginning, there are still many problems that need to be solved (problems that are my fault or not). I tried to add several features in MCreator to solve as many problems as possible, but some elements still need to be improved before an official addition.

TL;DR: No, nobody desired to continue it for the moment.

**Comment by Goldorion** [comment]
Nobody decided to take it back for the moment. Nerdy didn't want to continue when we talked about my plugins (at least not soon). Concerning Klemen, I already asked him multiple times if there was a possibility to make it official in some way, but the answer always was the same. The amount of active contributors for the core is not high enough to justify the addition of another big generator. In addition, I was the only one in the team of main contributors to know and work with Fabric, so Klemen would need to learn Fabric before taking it back. Another reason was also the fact the generator, even though I worked hard on it and improved it quite a lot since the beginning, there are still many problems that need to be solved (problems that are my fault or not). I tried to add several features in MCreator to solve as many problems as possible, but some elements still need to be improved before an official addition.

The fact that nobody would continue it the day I stopped was my biggest fear because I always struggled to get and keep a small team of people helping me with its development. I always made 95-99% of the job myself. I tried many times to get and keep at least one person to help me, but the person always ended up leaving after some time, which I completely understand. I know this is a very big job and it requires a lot of effort and time. I tried to continue updating it for quite some time even if I lost the motivation because I knew that if I was stopping, the generator would also stop being updated. However, I didn't want to continue for the reasons I already explained.

Anyway, the short answer is no because nobody had the desire to continue its development for the moment.

Edit: If nobody updates it and I have some motivation to work on a small update/porting version, I may shadowdrop one in the future though (same thing for other plugins and/or features for MCreator's core). I don't close myself to stop entirely MCreator forever. I just want to quit the "One of the most active and prolific users of the community dedicating **a lot** of time to MCreator."

**Comment by Realbizzy** [comment]
> Nobody decided to take it back for the moment. Nerdy didn't want to continue when we talked about my plugins (at least not soon). Concerning Klemen, I already asked him multiple times if there was a possibility to make it official in some way, but the answer always was the same. The amount of active contributors for the core is not high enough to justify the addition of another big generator. In addition, I was the only one in the team of main contributors to know and work with Fabric, so Klemen would need to learn Fabric before taking it back. Another reason was also the fact the generator, even though I worked hard on it and improved it quite a lot since the beginning, there are still many problems that need to be solved (problems that are my fault or not). I tried to add several features in MCreator to solve as many problems as possible, but some elements still need to be improved before an official addition.
> 
> The fact that nobody would continue it the day I stopped was my biggest fear because I always struggled to get and keep a small team of people helping me with its development. I always made 95-99% of the job myself. I tried many times to get and keep at least one person to help me, but the person always ended up leaving after some time, which I completely understand. I know this is a very big job and it requires a lot of effort and time. I tried to continue updating it for quite some time even if I lost the motivation because I knew that if I was stopping, the generator would also stop being updated. However, I didn't want to continue for the reasons I already explained.
> 
> Anyway, the short answer is no because nobody had the desire to continue its development for the moment.
> 
> Edit: If nobody updates it and I have some motivation to work on a small update/porting version, I may shadowdrop one in the future though (same thing for other plugins and/or features for MCreator's core). I don't close myself to stop entirely MCreator forever. I just want to quit the "One of the most active and prolific users of the community dedicating **a lot** of time to MCreator."


Well Gold, after so long someone FINALLY did it!, here! https://mcreator.net/plugin/110257/fabric-generator-1201

**Comment by Goldorion** [comment]
I saw the plugin when it got published. However, I don't think we can consider it continues my work, as this user seems to do all over again what I did (and maybe expand, I didn't go for the details) instead of updating/upgrading my existing plugin. However, this is nice to see someone taking the lead for a Fabric Generator! It takes a lot of motivation (or a lot of stupidity... not sure about the answer 🤣)

**Comment by Realbizzy** [comment]
> I saw the plugin when it got published. However, I don't think we can consider it continues my work, as this user seems to do all over again what I did (and maybe expand, I didn't go for the details) instead of updating/upgrading my existing plugin. However, this is nice to see someone taking the lead for a Fabric Generator! It takes a lot of motivation (or a lot of stupidity... not sure about the answer 🤣)

yeah lol ig, it does seam stressful after all
